bugbug/tools/code_review.py

```diff
@@ -1074,11 +1082,17 @@ def run(self, patch: Patch) -> list[InlineComment] | None:
             return_only_outputs=True,
         )["text"]
 
+        if self.verbose:
+            print(f"\u001b[33;1m\033[1;3m{output_summarization}\u001b[0m")
+
         if self.function_search is not None:
             line_code_list = self.further_context_chain.run(
                 patch=formatted_patch, summarization=output_summarization
             ).split("\n")
 
+            if self.verbose:
+                print(f"\u001b[33;1m\033[1;3m{line_code_list}\u001b[0m")
```

Comment:
We may rap the print formatting in a function to make it more readable.

Reply:
Done!